gui/package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "frdr-crypto",
-  "version": "0.2.0",
-  "homepage": "https://github.com/jza201/frdr-secure-data",
+  "name": "frdr-encryption-client",
+  "version": "0.3.0",
+  "homepage": "https://github.com/frdr-dfdr/frdr_encryption_client",
   "main": "main.js",
   "description": "A local application to encrypt and decrypt data.",
   "license": "MIT",
```

Comment:
I think we don't have an answer back yet on the license for this code.  Better to put "CUSTOM" instead of MIT for now.

Reply:
Updated.